src/main/java/com/puppycrawl/tools/checkstyle/api/TokenTypes.java

```diff
@@ -1785,6 +1785,27 @@
     /**
      * Literal {@code while} in do-while loop.
      *
+     * <p>For example:</p>
+     * <pre>
+     * do {
+     *
+     * } while (a &gt; 0);
+     * </pre>
+     * <p>parses as:</p>
+     * <pre>
+     * |--LITERAL_DO -&gt; do
```

Comment:
please remove `|` , it does not lead to anywhere

Reply:
Requested changes done, kindly review the changes please @romani